spec/fixtures/cassettes/push_notification_devices_destroy_many.json

```diff
@@ -0,0 +1 @@
+{"http_interactions":[{"request":{"method":"post","uri":"http://agent%40zendesk.com:123456@support.zd-dev.com/api/v2/push_notification_devices/destroy_many","body":{"encoding":"UTF-8","base64_string":"eyJwdXNoX25vdGlmaWNhdGlvbl9kZXZpY2VzIjpbImZvbyIsImJhciJdfQ==\n"},"headers":{"Accept":["application/json"],"Accept-Encoding":["gzip;q=1.0,deflate;q=0.6,identity;q=0.3"],"User-Agent":["ZendeskAPI API 1.6.1"],"Content-Type":["application/json"]}},"response":{"status":{"code":200,"message":"OK"},"headers":{"Content-Type":["application/json; charset=UTF-8"],"Content-Length":["1"],"Connection":["keep-alive"],"Status":["200 OK"],"X-Zendesk-Api-Version":["v2"],"X-Frame-Options":["SAMEORIGIN"],"X-Zendesk-Origin-Server":["ASSET7006-jnarvaez.local"],"X-Ua-Compatible":["IE=Edge"],"Etag":["W/\"7215ee9c7d9dc229d2921a40e899ec5f\""],"Cache-Control":["max-age=0, private, must-revalidate"],"X-Request-Id":["37471420eda797afa6b57400730acef7"],"X-Runtime":["1.143524"],"X-Zendesk-Request-Id":["9117c96115f3186ff543"],"X-Powered-By":["Phusion Passenger 4.0.45"],"Date":["Fri, 27 Mar 2015 11:32:25 UTC"],"Server":["nginx/1.4.4 + Phusion Passenger 4.0.45"]},"body":{"encoding":"UTF-8","base64_string":"IA==\n"},"http_version":null},"recorded_at":"Fri, 27 Mar 2015 11:32:25 GMT"}],"recorded_with":"VCR 2.9.3"}
```

Comment:
Hey @steved could you elaborate more about what do you mean. What I did was to point to one of our test servers and run the tests in order to record the VCR cassettes.